bigchaindb/README.md

```diff
@@ -1,8 +1,8 @@
 # Overview
 
-A high-level description of the files and subdirectories of BigchainDB. Heavily used external dependencies include [`multipipes`](https://github.com/bigchaindb/multipipes) and [`bigchaindb-common`](https://github.com/bigchaindb/bigchaindb-common).
+A high-level description of the files and subdirectories of BigchainDB.
```

Comment:
Well, @rhsimplex authored this, and perhaps may share his thoughts. This seems to me a question of opinions here.